packages/sdks/web-js-sdk/src/enhancers/helpers/index.ts

```diff
@@ -116,11 +126,16 @@ export const getUserFromResponse = async (
 export const isDescopeBridge = () =>
   typeof window !== 'undefined' && !!window['descopeBridge'];
 
-export const isLocalStorage = typeof localStorage !== 'undefined';
+export const isLocalStorage =
+  typeof customStorage !== 'undefined' || typeof localStorage !== 'undefined';
 
 export const setLocalStorage = (key: string, value: string) =>
-  isLocalStorage && localStorage?.setItem(key, value);
+  (customStorage || localStorage)?.setItem?.(key, value);
 export const getLocalStorage = (key: string) =>
-  isLocalStorage && localStorage?.getItem(key);
+  (customStorage || localStorage)?.getItem?.(key);
 export const removeLocalStorage = (key: string) =>
```

Comment:
there are other places that need to be considered
<img width="1005" height="618" alt="image" src="https://github.com/user-attachments/assets/b388b24a-0ad8-4447-9b16-888bed61f947" />


maybe I missed it, but I didn't see that those places also uses the custom storage

Reply:
Done